x-pack/plugins/timelines/public/mock/t_grid.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TGridIntegratedProps } from '../components/t_grid/integrated';
+import { mockBrowserFields, mockDocValueFields } from './browser_fields';
+import { mockDataProviders } from './mock_data_providers';
+import { ColumnHeaderOptions, TimelineId } from '../../common';
+import { mockIndexNames, mockIndexPattern } from './index_pattern';
+
+const columnHeaders: ColumnHeaderOptions[] = [
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: 'Status',
+    id: 'kibana.rac.alert.status',
+    initialWidth: 79,
+    category: 'kibana',
+    type: 'string',
+    aggregatable: true,
+    actions: {
+      showSortAsc: {
+        label: 'Sort A-Z',
+      },
+      showSortDesc: {
+        label: 'Sort Z-A',
+      },
+    },
+    defaultSortDirection: 'desc',
+    display: {
+      key: null,
+      ref: null,
+      props: {
+        children: {
+          key: null,
+          ref: null,
+          props: {
+            children: 'Status',
+          },
+          _owner: null,
+        },
+      },
+      _owner: null,
+    },
+    isSortable: true,
+  },
+  {
+    columnHeaderType: 'not-filtered',
+    displayAsText: 'Triggered',
+    id: 'kibana.rac.alert.start',
```

Comment:
same here

Reply:
i think it is important for us to always use @kbn/rule-data-utils, so please change it before merging it